<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptWidget</class>
 <widget class="QWidget" name="ScriptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="Python2">
    <attribute name="title">
     <string>Python2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Python3">
    <attribute name="title">
     <string>Python3</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Lua">
    <attribute name="title">
     <string>Lua</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>381</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>441</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>当前选择脚本:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
